src/libraries/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CurrentUserOnly.Windows.cs

```diff
@@ -22,14 +22,10 @@ public class TestAccountImpersonator : IDisposable
         public TestAccountImpersonator()
         {
             string testAccountPassword;
-            using (RandomNumberGenerator rng = new RNGCryptoServiceProvider())
-            {
-                var randomBytes = new byte[33];
-                rng.GetBytes(randomBytes);
+            var randomBytes = RandomNumberGenerator.GetBytes(33);
```

Comment:
Improper use of `var`:

```suggestion
            byte[] randomBytes = RandomNumberGenerator.GetBytes(33);
```

Reply:
Thanks for fixing that, @bartonjs.